curvefs/test/metaserver/inode_manager_test.cpp

```diff
@@ -72,7 +73,9 @@ class InodeManagerTest : public ::testing::Test {
         auto inodeStorage = std::make_shared<InodeStorage>(
             kvStorage_, nameGenerator, 0);
         auto trash = std::make_shared<TrashImpl>(inodeStorage);
-        manager = std::make_shared<InodeManager>(inodeStorage, trash);
+        map_ = std::make_shared<FileType2InodeNumMap>();
```

Comment:
`map` can be a data member rather than a pointer, and give a more meaningful name